includes/class-wcdn-print.php

```diff
@@ -543,16 +543,29 @@ private function populate_orders() {
 				$order = $post;
 
 				$wdn_order_id = ( version_compare( get_option( 'woocommerce_version' ), '3.0.0', '>=' ) ) ? $order->get_id() : $order->id;
+				// Allow admins to view all orders.
+				if ( current_user_can( 'administrator' ) ) {
```

Comment:
We can the capability as current_user_can( 'manage_woocommerce' ) so that users who can manage WC orders will be able to check invoices.

Reply:
done.